static/styles/administration/dataprivacy.scss

```diff
@@ -6,15 +6,15 @@
     /* insert custom css for the forms here */
     .show-invalid{
         input:invalid:not([type="checkbox"]):not([type="radio"]){
-            box-shadow: 0 0 5px $primaryColor;
+            box-shadow: 0 0 5px 1px $colorDanger;
```

Comment:
looks great. is it correct to have 4 numerical values here (box-shadow) instead of 3 like before?
does that solution work for all themes right now?

Reply:
Yes that is correct. The new number is the spread-radis ( https://developer.mozilla.org/de/docs/Web/CSS/box-shadow )
And it works for all themes because I added the color in he global color file that is the same for all themes.